src/util/tpool/fd_tpool.cxx

```diff
@@ -1,4 +1,5 @@
 #include "fd_tpool.h"
+#include <malloc.h>
```

Comment:
Not sure why these tpool changes are in here?

Reply:
Because this became an integration branch trying to get the test to pass.  We are currently talking about what to do with this stuff right now.. tbh

Reply:
reverted